src/framework/theme/components/component-size.ts

```diff
@@ -1 +1 @@
-export type NbComponentSize = '' | 'tiny' | 'small' | 'medium' | 'large' | 'giant';
+export type NbComponentSize = 'tiny' | 'small' | 'medium' | 'large' | 'giant';
```

Comment:
so, no empty status after all?

Reply:
Yep. No more empty values for size, status or shape. Component which could has unset value, should add `''` as possible value. Like this:
`status: '' | NbComponentStatus`